### DISCUSSION TITLE
怎么更新sing-box核心呢？

### DISCUSSION THREAD

**Discussion by evenz82** [UIforFreedom/UIF#56] [Q&A] [answered]
怎么更新sing-box核心呢？

**Comment by UIforFreedom** [comment, marked as answer]
在设置里面选择自动更新即可。 不需要你自己特别关注，更新

Sent from [Proton Mail](https://proton.me/mail/home) for iOS

2024 年 12 月 7 日星期六 22:16, evenz82 ***@***.***(mailto:2024 年 12 月 7 日星期六 22:16, evenz82 <<a href=)> 来信：

**Comment by evenz82** [comment]
<img width="230" alt="Snipaste_2024-12-07_23-29-34" src="https://github.com/user-attachments/assets/f458be37-e90c-4fb5-9b78-9f5b73e66cab">
<img width="118" alt="Snipaste_2024-12-07_23-29-56" src="https://github.com/user-attachments/assets/70d5fbb6-06e3-4104-974d-1c9dba8dab7b">
似乎只有面板会更新，sing-box核心一直不更新。现在最新的版本是1.10.3了

**Comment by UIforFreedom** [comment]
是的，我很久没更新内核版本了。 最近 sing box 大改，不稳定

Sent from [Proton Mail](https://proton.me/mail/home) for iOS

2024 年 12 月 7 日星期六 23:32, evenz82 ***@***.***(mailto:2024 年 12 月 7 日星期六 23:32, evenz82 <<a href=)> 来信：
